pom.xml

```diff
@@ -1010,6 +1030,20 @@
                 <groupId>com.google.cloud.bigdataoss</groupId>
                 <artifactId>gcsio</artifactId>
```

Comment:
If you are using GCS connector only through HDFS/HCFS API you can depend on [shaded JAR](https://repo1.maven.org/maven2/com/google/cloud/bigdataoss/gcs-connector/hadoop2-2.1.2) instead (`<classifier>shaded</classifier>`). This way you will have only one dependency instead of 4 dependencies now.